Sources/AblyChat/Messages.swift

```diff
@@ -1,10 +1,12 @@
 import Ably
 
+public typealias RealtimeChannel = any(ARTRealtimeChannelProtocol & Sendable)
+
 public protocol Messages: AnyObject, Sendable, EmitsDiscontinuities {
     func subscribe(bufferingPolicy: BufferingPolicy) async -> MessageSubscription
     func get(options: QueryOptions) async throws -> any PaginatedResult<Message>
     func send(params: SendMessageParams) async throws -> Message
-    var channel: ARTRealtimeChannelProtocol { get }
```

Comment:
I’m curious of the motivation for ead2b9b0cb3ea097a571686f76fec016f3d001b9 — why do these `channel` properties need to be marked as `Sendable` in the public API?

Reply:
@lawrence-forooghian because of this error:
<img width="1118" alt="Screenshot 2024-09-26 at 21 13 21" src="https://github.com/user-attachments/assets/f47cf03f-5ef7-442e-b00a-75d7154d7aed">

Reply:
OK

- the commit message for ead2b9b0cb3ea097a571686f76fec016f3d001b9 is misleading; it suggests that you've just added a typealias, but you've actually changed the public API in a meaningful way (requiring `Sendable` conformance) so please fix that and also add the motivation
- please undo the unnecessary switching of order in `RoomReactions.swift`

Reply:
https://github.com/ably-labs/ably-chat-swift/pull/34/commits/0ec4b7bc474920c891bcb3e8bf64f0f9f5337148